include/seqan3/std/ranges

```diff
@@ -392,21 +392,21 @@ namespace std::ranges::view
  */
 inline constexpr auto all = ::ranges::view::all;
 
-} // namespace std::ranges::view
+} // namespace std::ranges::views
```

Comment:
here and line 406 why views?

Reply:
hard to explain, but should be this way.